minichain/p2p.py

```
@@ -228,6 +228,21 @@ def _validate_block_payload(self, payload):
             for tx_payload in payload["transactions"]
         )
 
+    def _validate_chain_request(self, payload):
+        if not isinstance(payload, dict):
+            return False
+        return True
+
+    def _validate_chain_response(self, payload):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_validate_chain_response`

Add return type annotation: `bool`

(ANN202)